packages/code-du-travail-frontend/src/outils/common/Wizard.js

```diff
@@ -68,27 +68,25 @@ function Wizard({
         }}
         onSubmit={handlePageSubmit}
       >
-        {({ handleSubmit, form, invalid }) => {
-          return (
-            <>
-              <form onSubmit={handleSubmit}>
-                {Rules && (
-                  <Rules values={form.getState().values} dispatch={dispatch} />
-                )}
-                <StepItems activeIndex={stepIndex} items={stepItems} />
-                <StepWrapper narrow noPadding>
-                  <Step form={form} />
-                </StepWrapper>
-                <PrevNextBar
-                  onPrev={prevStep}
-                  nextVisible={nextVisible}
-                  disabled={invalid}
-                  previousVisible={previousVisible}
-                />
-              </form>
-            </>
-          );
-        }}
+        {({ handleSubmit, form, invalid }) => (
```

Comment:
explicit return is handier since it's easier to add console for debug sometimes rather that edit / reformat
so can you revert it, please  ?

Reply:
I disagree with that :/ To me we should enforce with an eslint rule that arrow functions with no scoped variables / console.log should have implicit, and not explicit return. Otherwise we should ban the implicit return everywhere. But I don't like having the two and do guesstimate.

Reply:
well to me it's not really important to have both but yeah we could ban implicit for return function that not fit on oneline ;)

Reply:
I'll add this to my todo  !

Reply:
It is important to me because these kind of refactor on such opinionated details is a pure loss of time to me and I want to avoid that by all means.
It must be my 50th PR and today, for I don't know which reason, when I have really other things to be concerned about, I mean real things, it is a topic of discussion.
So OK, I'll revert it back, but I don't want to ever talk about it anymore.

Reply:
> It is important to me because these kind of refactor on such opinionated details is a pure loss of time to me and I want to avoid that by all means.

I agree, I propose to not do it and leave it. I think don't do any comment about style (return vs implicit return vs function) as It's up to personnal feelings and it has no impact on code execution.  Doing that in PR, add more noise, take more (code + review) for no impact that's why I think we should avoid that kind of "refactor". 
Does it sounds ok to you  ?

Reply:
I must have added a console log and then when removing it simply changed the "style" without even realizing I changed something ! But yeah it sounds ok !